src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -212,26 +196,15 @@ static SimplexResult OPT_TryToCallSimplex(const OptimizationOptions& options,
     ProblemeAResoudre->ExistenceDUneSolution = Probleme.ExistenceDUneSolution;
     if (ProblemeAResoudre->ExistenceDUneSolution != OUI_SPX && PremierPassage)
     {
-        if (ProblemeAResoudre->ExistenceDUneSolution != SPX_ERREUR_INTERNE)
```

Comment:
We used to test `ExistenceDUneSolution != SPX_ERREUR_INTERNE`.
But `ExistenceDUneSolution == SPX_ERREUR_INTERNE` is impossible : indeed, ExistenceDUneSolution is given by **ortools** now, which does not know value **SPX_ERREUR_INTERNE**.